Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -141,6 +141,10 @@ import BraintreeCore
         if let userAuthenticationEmail, !userAuthenticationEmail.isEmpty {
             checkoutParameters["payer_email"] = userAuthenticationEmail
         }
+        
+        if let userPhoneNumber, !userPhoneNumber.isEmpty {
+            checkoutParameters["phone_number"] = userPhoneNumber
+        }
```

Comment:
Can we move this logic to the `BTPayPalRequest` directly so we don't need to set it both here and in the vault request? Not sure what level it needs to be set at so may not be possible.

Reply:
yep, addressed in 53255318f9fa40fa7fdc6ba72359ba04ac75a495